.sbtopts

```diff
@@ -1,3 +1,3 @@
--J-Xms2g
--J-Xmx4g
--J-XX:MaxMetaspaceSize=512m
+-J-Xms4g
+-J-Xmx8g
+-J-XX:MaxMetaspaceSize=1024m
```

Comment:
GitHub runners only have 7 GB of RAM, and we need to leave memory for forked JVMs / Node.js / Native processes.